sprocs/update_group_pogil_role.sql

```diff
@@ -0,0 +1,19 @@
+DROP FUNCTION IF EXISTS update_group_pogil_role(enum_pogil_role,bigint,bigint);
+CREATE OR REPLACE FUNCTION
+    update_group_pogil_role (
```

Comment:
Please name sprocs starting with the table name (in this case `group_users`). I think this sproc should be `group_users_update_pogil_role`.